### PR TITLE
add pyglider

### PR DESCRIPTION
**Pull request by jklymak**
I think I did this correctly - preferable as a conda recipe because it depends on netcdf4 and dask, and both are better supplied by conda than pip.  

Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/5eddbd7fc9d1502169089da06c3688d9759be978/recipes/example/meta.yaml#L64-L73) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- N/A If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- N/A When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by jklymak** [comment]
@conda-forge/help-python - I think this is ready for review, but please let me know if there are any errors..

File: recipes/pyglider/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+  commands:
+    - pip check
+  requires:
+    - pip
+
+about:
+  home: https://pyglider.readthedocs.io
+  summary: Glider data to netCDF translation in python
+  license: Apache-2.0
+  license_file: LICENSE
```
Comment (BastianZim) [inline recipes/pyglider/meta.yaml:43]:
```suggestion
  license_file: LICENSE
  dev_url: https://github.com/c-proof/pyglider
```

**Review by BastianZim — CHANGES_REQUESTED** [review]
Just one small cosmetic thing, then this is ready to go 🚀

**Comment by jklymak** [comment]
@conda-forge/help-python Added the requested field, should be good to go again!  Thanks,

**Comment by jklymak** [comment]
Thanks for your help @BastianZim

**Pull request merged**